sortedm2m/fields.py

```diff
@@ -212,17 +212,23 @@ class SortedManyToManyField(_ManyToManyField):
     '''
     Providing a many to many relation that remembers the order of related
     objects.
-
+    
```

Comment:
you are adding spaces and changing a line that doesn't need to be changed, try to revert this change (insert a blank line without space, commit and then squash/fixup).